### PR TITLE
Add SMB file sharing tutorial.

### PR DESCRIPTION
**Pull request by eadamsintel**
Signed-off-by: Adams <eric.adams@intel.com>

### CODE REVIEW

**Comment by klynnrif** [comment]
Hello @eadamsintel - I have added in edits for you to approve or comment on. Thanks!

**Comment by eadamsintel** [comment]
Thanks.  I made one tiny change to your changes that I think helps a sentence read more clearly.

Thanks
Eric

File: source/clear-linux/tutorials/smb/smb.rst [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+#.	Follow the :ref:`install-on-target` guide to install |CL|.
+#.	Add the storage-utils bundle, which includes the Samba binaries, after |CL|
+	is setup. The os-clr-on-clr bundle also includes the Samba binaries.
+
+	.. code-block:: console
+
+		$sudo swupd bundle-add storage-utils
+
+#.	Create a configuration file at :file:`/etc/samba/smb.conf` and add the following
+	(change as needed). The following configuration has two simple examples.
```
Comment (rcaballeromx) [inline source/clear-linux/tutorials/smb/smb.rst]: @eadamsintel Can you please update whether this PR is ready to be merged. I found this note an stopped my review.

**Comment by eadamsintel** [comment]
Yes it is ready to be merged.

**Comment by rcaballeromx** [comment]
@eadamsintel: I need to make several edits. Is it ok for me to push them to your branch?

**Comment by eadamsintel** [comment]
This is approved. Is there anything else I need to do?

**Comment by MCamp859** [comment]
@eadamsintel  I have pushed changes into your branch. Please have a look and let me know if you have questions. Thanks!

File: source/clear-linux/tutorials/smb/smb.rst [hunk 2]
```diff
@@ -1,31 +1,56 @@
...
-        The first example demonstrates sharing a folder to a specific user and
-        the second example demonstrates sharing a folder to anyone. The examples
-	assume that a user account named "clearlinuxuser" already exists.
-	If "valid users" is not specified as shown, then anyone with a user account
-	on this machine and with their Samba password already set can access the files.
-        However, that account is only able to access files and folders that the
-        account has permission to on the system. Use :command:`chown`
-	or :command:`chmod` to change either the owner of the file or the
-	permissions to allow other users to access that file.
+		The os-clr-on-clr bundle also includes the Samba binaries.
```
Comment (MCamp859) [inline source/clear-linux/tutorials/smb/smb.rst]: Question: is more detail needed about the `os-clr-on-clr` bundle? Or are we saying "If you installed the `os-clr-on-clr` bundle then you don't need this step"? (Line 40 is a Note in rendered output but it looks odd here.)

**Comment by MCamp859** [comment]
@eadamsintel  Based on [PR 93's](https://github.com/clearlinux/clear-linux-documentation/pull/93) comments, the first step should be "sudo -s" and then the rest of the commands are OK without it. I will confirm with Rodrigo.

**Comment by eadamsintel** [comment]
I am ok with publishing then. Everything else looks ok from what I can tell.

**Comment by rcaballeromx** [comment]
@eadamsintel Understood, I also solved the merge conflict and fixed the toctree directive for the content to be included properly.

**Pull request merged**